c/tskit/tables.h

```diff
@@ -625,6 +625,13 @@ typedef struct _tsk_segment_t {
     tsk_id_t node;
 } tsk_segment_t;
 
+typedef struct {
+    tsk_size_t num_segments;
+    double total_span;
```

Comment:
What's this for?

Reply:
(the `total_span` attribute)

Reply:
The sum of the ``right - left`` values in the segments. I figured this was a handy summary which would be useful when we optionally skip storing the segments.